docs/docs/get_around_unsupported_features.md

```diff
@@ -0,0 +1,25 @@
+# Get around unsupported features.
+
+There is no algorithm to get around unsupported features, in most scenarios it is necessary to understand what the piece of code is doing. There are some unsupported features that are impossible to rewrite right now, understanding what it is expected to do when translated to Cairo may indicate that the logic can just be removed.
+
+## Unsupported features that can not be resolved.
+
+- Function gasleft(): No way to go around it, but in most cases won't be needed.
+- Try/Catch: Rewrite the logic using solidity
+- Nested tuple expressions: Should be rewritten using solidity to flat it
+- TypeName expressions: No equivalent in Cairo
+- msg.value, msg.sig, msg.data: No equivalent in Cairo so far
+- tx.gasprice, tx.origin: No equivalent in Cairo so far
+- Member access of block(block.gaslimit, block.chainid): No equivalent in Cairo. Most of the time won't be needed.
+- User defined Errors: No equivalent in Cairo for now.
+- Receive function: StarkNet doesn't have a native currency for the protocol so should be rewritten using other approach
+- Function call options e.g x.f{gas: 10000}(arg1): Most of the options passed are not compatible with Cairo so in most cases can be just removed
```

Comment:
Some bullet points have a dot at the end while others don't. Please put a dot at the end of each bullet point